Mainboard/README.md

```diff
@@ -254,3 +254,28 @@ as "JST SH", and has a 1.0 mm pitch. The same connector is used for the Speakers
 | 2   | FAN_SPEED |
 | 3   | FAN_PWM   |
 | 4   | GND       |
+
+## RTC Battery Interface
+
+NOTE: The mainboard will charge this battery, so a rechargable ML1220 battery must be used.
+CR2032 is not okay because it cannot be charged.
+
+### Onboard RTC Battery
+
+Framework 13 Intel Core 11th through 13th Generation use an onboard coin cell battery as RTC.
+These systems ship with a battery that keeps BIOS time, settings and configuration.
+
+We use the Lotes LOTES_AAA-BAT-046-K01 connector.
+
+### RTC Battery Cable Connector
+
+On AMD Platforms and starting with Framework 13 Intel Core Ultra Series 1, no RTC battery is included anymore.
+Settings are kept alive by the normal system battery. The only case where an
+RTC battery is needed, is when the system is used in standalone mode (without a battery).
+
+Cvilux CVILU_CI4402M1HRC-NH connector for the RTC Battery.
+
+| Pin | Signal    |
+|-----|-----------|
+| 1   | +RTCBATT  |
+| 2   | GND       |
```

Comment:
The pin numbers might not be obvious, how about we hint at left vs right with a drawing, or some kind of tip to look (or probe) for the ground plane.

Reply:
1 left, 2 right.